did-sdk/src/ion/keypairs.ts

```diff
@@ -0,0 +1,35 @@
+import { LdKeyPairInstance } from '@transmute/ld-key-pair'
+import { Ed25519KeyPair } from '@transmute/did-key-ed25519'
+import { Secp256k1KeyPair } from '@transmute/did-key-secp256k1'
+
+export type KeyType = 'ed25519' | 'secp256k1'
+export type generateFromSeedOptions = { secureRandom: () => Buffer }
```

Comment:
will the `Buffer` type work alright in browsers? does this allow for Uint8array/array buffer to be returned?

Reply:
We'll need to do a pass on all of this for browsers. I figured I'd wait till I have an environment using it so I can do the testing, but I'll see about switching to WebCrypto now